packages/integration-testsuite/src/httpServerTests.ts

```diff
@@ -2060,6 +2061,33 @@ export function defineIntegrationTestSuiteHttpServerTests(
           expect(res.body.errors[0].message).toEqual(expected);
         });
       });
+
+      it('uses stringifyResult parameter', async () => {
```

Comment:
We can just remove this test altogether with the existence of the other test. It looks like there are a handful of snapshot files that were generated due to using `toMatchSnapshot` instead of `toMatchInlineSnapshot` - we can delete those too.

Reply:
on it.